net/ghttp/ghttp_request_param.go

```diff
@@ -30,9 +30,9 @@ import (
 )
 
 const (
-	parseTypeRequest = 0
-	parseTypeQuery   = 1
-	parseTypeForm    = 2
+	parseTypeRequest = iota
```

Comment:
Is it necessary using `iota` instead of explicit number definition?

Reply:
close